platform/ios/src/MGLMapboxEvents.h

```diff
@@ -8,6 +8,7 @@ NS_ASSUME_NONNULL_BEGIN
 + (nullable instancetype)sharedInstance;
 
 + (void)setupWithAccessToken:(NSString *)accessToken;
++ (void)setAccessToken:(NSString *)accessToken;
```

Comment:
I find this very confusing (two similarly named methods taking the same parameter).

I'm wondering if we should have a `dispatch_queue` and use `dispatch_group` to coordinate initializing the `MMEEventsManager`, and sending of the events until everything is ready.

Reply:
Fair; this initial change was really only meant as a conversation starter (an issue probably would have been more appropriate 🤷‍♂). I like the idea of the dispatch.